src/management-system-v2/prisma/migrations/20240908215515_added_process_artifact_table/migration.sql

```diff
@@ -0,0 +1,223 @@
+-- CreateEnum
+CREATE TYPE "ProcessType" AS ENUM ('Process', 'Template', 'Projects');
```

Comment:
We don't use templates anymore, I think. We could leave this as a normal string, so we can make changes without changing the DB schema.